packages/sit-onyx/src/components/OnyxNotifications/useNotification.ts

```diff
@@ -99,7 +102,10 @@ export const useNotification = () => {
     () => {
       return {
         notifications: computed(() => []),
-        show: logWarning,
+        show: () => {
+          logWarning();
+          return 0;
```

Comment:
```suggestion
          return -1;
```

Would prefer to return -1 here so its clearer that something is used wrong in this case :)